src/unstakeAg/index.ts

```diff
@@ -855,25 +859,29 @@ export class UnstakeAg {
     }
 
     const v0ExchangeReturn = exchangeReturn as ExchangeReturnV0;
-    const addressLookupTableAccounts = await Promise.all(
-      v0ExchangeReturn.unstakeTransaction.message.addressTableLookups.map(
-        async (lookup) =>
-          new AddressLookupTableAccount({
-            key: lookup.accountKey,
-            state: AddressLookupTableAccount.deserialize(
-              await this.connection
-                .getAccountInfo(lookup.accountKey)
-                .then((res) => res!.data),
-            ),
-          }),
+    const pks = [
+      LUT_PK.toString(),
+      ...v0ExchangeReturn.unstakeTransaction.message.addressTableLookups.map(
+        (lookup) => lookup.accountKey.toString(),
       ),
+    ];
+    const accountInfos = await chunkedGetMultipleAccountInfos(
+      this.connection,
+      pks,
     );
-    const lut = await this.connection
-      .getAddressLookupTable(LUT_PK)
-      .then((res) => res.value);
-    const luts = lut
-      ? [lut, ...addressLookupTableAccounts]
-      : addressLookupTableAccounts;
+    const luts = accountInfos
+      .map((acc, index) => {
+        if (acc) {
+          return new AddressLookupTableAccount({
+            key: new PublicKey(pks[index]),
+            state: AddressLookupTableAccount.deserialize(acc.data),
+          });
+        }
+
+        return null;
+      })
+      .filter(Boolean) as AddressLookupTableAccount[];
```

Comment:
for some reason i have to type it `as` otherwise it ignores filter and type as `(AddressLookupTableAccount | null)[]`